### ISSUE TITLE
Empty logs during model.fit()

### ISSUE THREAD

**Issue by woahthere2** [tensorflow/tensorflow#60719]
[Issue_Empty_logs_during_model.fit().pdf](https://github.com/tensorflow/tensorflow/files/11584239/Issue_Empty_logs_during_model.fit.pdf)

**Comment by woahthere2** [comment]
TensorFlow version: 2.11.0 
Python version: 3.10.10 
GPU/CPU: GPU T4 x 2 on Kaggle

Additional steps taken:
Executed tf.config.run_functions_eagerly(True) , but the issue persists.
Confirmed all image sizes are (128, 128, 3) and model.input_shape is (None, 128, 128, 3) . 
Validated that the datasets are not empty by running the following code:
```for images, masks in val_dataset.take(1): 
sample_image, sample_mask = images[31], masks[31] 
display([sample_image, sample_mask])```
The displayed images and masks are as expected.

**Comment by tilakrayal** [comment]
@woahthere2,
I request you to take a look at this [issue](https://github.com/keras-team/keras/issues/16202) where a similar feature has been proposed and it is still open. Also I request to follow the similar feature which has been proposed to have the updates on the similar issue. Thank you!
